ledger/sandbox/BUILD.bazel

```diff
@@ -108,6 +108,17 @@ da_scala_binary(
     ],
 )
 
+da_scala_binary(
+    name = "sandbox-ephemeral-h2database",
+    main_class = "com.digitalasset.platform.sandbox.persistence.EphemeralH2DatabaseSandboxMain",
```

Comment:
We don't need a dedicated class for an "ephemeral H2 sandbox", because starting it with an H2 in-memory jdbc url will do the trick.

Reply:
Thanks, done.